nmz/cli/inspectors/proc.go

```diff
@@ -74,6 +77,18 @@ func (cmd procCmd) Run(args []string) int {
 		if _procFlags.Cmd != "" {
 			args := strings.Split(_procFlags.Cmd, " ")
 			cmd := exec.Command(args[0], args[1:]...)
+
+			if _procFlags.Stdout == "" {
+				cmd.Stdout = os.Stdout
```

Comment:
Keeping stdin as nil would be suitable. Input something via stdin would be confusing for iterated testing.